bindings/virtio/time.c

```diff
@@ -42,7 +42,7 @@ solo5_time_t solo5_clock_wall(void)
     if (use_pvclock)
         return pvclock_monotonic() + pvclock_epochoffset();
```

Comment:
```suggestion
        return pvclock_read_wall_clock();
```

Reply:
Yes, I will delete your commit. With PVH, a memory area is assigned to contains the real wall clock of the host system and we are allowed to read into it.